tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -304,6 +304,7 @@
   "DD_GIT_COMMIT_SHA": "commit_sha",
   "DD_TRACE_GIT_METADATA_ENABLED": "git_metadata_enabled",
   "DD_TRACE_ENABLED": "trace_enabled",
+  "DD_EXPERIMENTAL_APPSEC_STANDALONE_ENABLED": "experimental_appsec_standalone_enabled",
```

Comment:
Thanks for this, don't forget to make a PR to dd-go 🙂

Reply:
[Already merged 😄 ](https://github.com/DataDog/dd-go/pull/135833)